mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/XMLAttribute.java

```diff
@@ -44,11 +44,12 @@
     ADVERTISE_SOCKET(ProxyConfigurationResourceDefinition.Attribute.ADVERTISE_SOCKET),
     AUTO_ENABLE_CONTEXTS(ProxyConfigurationResourceDefinition.Attribute.AUTO_ENABLE_CONTEXTS),
     BALANCER(ProxyConfigurationResourceDefinition.Attribute.BALANCER),
-    CONNECTOR(ProxyConfigurationResourceDefinition.Attribute.CONNECTOR),
+    @Deprecated CONNECTOR("connector"),
```

Comment:
This should still reference the deprecated attribute alias.

Reply:
Fixed.